all-the-icons.el

```diff
@@ -142,19 +142,19 @@
     ("^LICENSE$"        all-the-icons-octicon "book"                    :height 1.0 :v-adjust 0.0 :face all-the-icons-blue)
     ("^readme"          all-the-icons-octicon "book"                    :height 1.0 :v-adjust 0.0 :face all-the-icons-lcyan)
 
-    ("\\.fish"          all-the-icons-alltheicon "terminal"             :face all-the-icons-lpink)
-    ("\\.zsh"           all-the-icons-alltheicon "terminal"             :face all-the-icons-lcyan)
-    ("\\.sh"            all-the-icons-alltheicon "terminal"             :face all-the-icons-purple)
+    ("\\.fish$"         all-the-icons-alltheicon "terminal"             :face all-the-icons-lpink)
+    ("\\.zsh$"          all-the-icons-alltheicon "terminal"             :face all-the-icons-lcyan)
+    ("\\.sh$"           all-the-icons-alltheicon "terminal"             :face all-the-icons-purple)
 
     ;; Config
-    ("\\.node"          all-the-icons-alltheicon "nodejs"               :height 1.0  :face all-the-icons-green)
+    ("\\.node$"         all-the-icons-alltheicon "nodejs"               :height 1.0  :face all-the-icons-green)
     ("\\.babelrc$"      all-the-icons-fileicon "babel"                  :face all-the-icons-yellow)
     ("\\.bashrc$"       all-the-icons-alltheicon "script"               :height 0.9  :face all-the-icons-dpink)
     ("\\.bowerrc$"      all-the-icons-alltheicon "bower"                :height 1.0 :v-adjust 0.0 :face all-the-icons-silver)
     ("^bower.json$"     all-the-icons-alltheicon "bower"                :height 1.0 :v-adjust 0.0 :face all-the-icons-lorange)
     ("\\.ini$"          all-the-icons-octicon "settings"                :v-adjust 0.0 :face all-the-icons-yellow)
-    ("\\.eslintignore"  all-the-icons-fileicon "eslint"                 :height 0.9  :face all-the-icons-purple)
-    ("\\.eslint"        all-the-icons-fileicon "eslint"                 :height 0.9  :face all-the-icons-lpurple)
+    ("\\.eslintignore$" all-the-icons-fileicon "eslint"                 :height 0.9  :face all-the-icons-purple)
+    ("\\.eslint$"       all-the-icons-fileicon "eslint"                 :height 0.9  :face all-the-icons-lpurple)
```

Comment:
pretty sure this isn't a mistake. `.eslint` can have multiple extensions.

Reply:
I reverted that change